config/locales/exceptions.en.yml

```diff
@@ -301,11 +345,55 @@ en:
         code: 'SM117'
         detail: 'A data integrity issue was encountered'
         status: 502
+      SM118:
+        <<: *external_defaults
+        code: 'SM118'
+        detail: 'It is not possible to move a message from the DRAFTS folder'
+        status: 422
       SM119: # 'Triage team does not exist' when trying to create new message
         <<: *external_defaults
         code: 'SM119'
         detail: 'Triage team does not exist'
         status: 422
+      SM120:
```

Comment:
I noticed that some of the exception types below don't have a status attribute. Is that intentional?

Reply:
If no status is defined it will default to a 400:
https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/common/exceptions/backend_service_exception.rb#L75